services/udp/config_test.go

```diff
@@ -29,6 +30,8 @@ batch-timeout = "10ms"
 		t.Fatalf("unexpected bind address: %s", c.BindAddress)
 	} else if c.Database != "awesomedb" {
 		t.Fatalf("unexpected database: %s", c.Database)
+	} else if c.RetentionPolicy != "awesomerp" {
```

Comment:
It will default to "", like all strings in Go. So then you'll end up with how the system is now, right?